README.md

```diff
@@ -18,28 +18,42 @@ This theme has been upgraded to `v4.0.0` to support the latest version of `Ghost
 
 ### Table of Contents
 
-* [Theme Features](#theme-features)
-* [Installation](#installation)
-* [Customizations](#customizations)
-    * [Number of Posts](#number-of-posts) 
-    * [Header Icon](#header-icon)
-    * [Favicons](#favicons)
-    * [Tags Overlay](#tags-overlay)
-    * [Google Analytics](#google-analytics)
-    * [Disqus Comments](#disqus-comments)
-    * [Masonry Grid](#masonry-grid-layout-beta)
-    * [Mailchimp](#mailchimp)
-    * [Toast Notifications](#toast-notifications)
-    * [Cover](#cover)
-    * [Links](#links)
-    * [Browser Compatibility Page](#browser-compatibility-page)
-    * [Custom Static Pages](#custom-static-pages)
-    * [Social Networks](#social-networks)
-    * [AMP Support](#amp-support)
-* [Developing and Contributing](#developing-and-contributing)
-    * [Building](#building)
-    * [Colors](#colors)
-    * [Preparing for Production](#preparing-for-production)
+- [Caffeine Theme ![Build Status](https://travis-ci.org/kelyvin/caffeine-theme) [![GitHub version](https://badge.fury.io/gh/kelyvin%2Fcaffeine-theme.svg)](https://github.com/kelyvin/caffeine-theme/releases)](#caffeine-theme--)
+  - [Introduction](#introduction)
+  - [Ghost Versions Support](#ghost-versions-support)
+    - [Table of Contents](#table-of-contents)
```

Comment:
Thanks for creating an updated new table of contents! However, since it looks autogenerated, there are some entries in here that make it more confusing. 

Remove this line and everything before this. The rest is fine.